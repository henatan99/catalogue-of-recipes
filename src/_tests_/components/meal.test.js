import React from 'react';
import { create } from 'react-test-renderer';

import Meal from '../../components/meal';

it('renders when no props passed', () => {
    const tree = create(<Meal />).toJSON;
    expect(tree).toMatchSnapshot();
});

it('renders when name props are passed', () => {
    const tree = create(<Meal name="chicken" image="image" author="author" />).toJSON; 
    expect(tree).toMatchSnapshot();
});
  
