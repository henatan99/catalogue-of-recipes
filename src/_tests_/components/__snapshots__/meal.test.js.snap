// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when name props are passed 1`] = `[Function]`;

exports[`renders when no props passed 1`] = `[Function]`;
